content/en/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -224,23 +273,30 @@ This will tell us 3 things:
 
 If the DNS and communication test passes, next you will need to check if the Pod has established secure channel communication with the domain.  To do this, again, `exec` into your Pod and run the `nltest.exe /query` command.
 
-```shell
-PS C:\> nltest.exe /query
+```PowerShell
+nltest.exe /query
+```
+
+Results in the following output:
+```PowerShell
```

Comment:
@perithompson please omit the “PowerShell” from this line - it's not something you type into a shell.